interface/forms/track_anything/report.js

```diff
@@ -14,9 +14,11 @@ function ta_report_getCheckedBoxes(chkboxName) {
   var checkboxes = document.getElementsByName(chkboxName);
   var checkedValue = [];
   // loop over them all
-  for (var i=0; i<checkboxes.length; i++) {
+  for (let i=0; i<checkboxes.length; i++)
```

Comment:
hi @dheero I think there is no need to provide extra space code format is already according to the standards.